src/v2/Apps/Partner/Routes/Works/index.tsx

```diff
@@ -1,5 +1,48 @@
 import React from "react"
+import { Box, GridColumns, Column } from "@artsy/palette"
+import ArtworkGrid from "v2/Components/ArtworkGrid"
+import { createFragmentContainer } from "react-relay"
+import { graphql } from "relay-runtime"
+import { Works_partner } from "v2/__generated__/Works_partner.graphql"
 
-export const WorksRoute: React.FC = () => {
-  return <div>Works</div>
+interface ArtworksProps {
+  partner: Works_partner
 }
+
+export const Works: React.FC<ArtworksProps> = ({ partner }) => {
+  const { artworksConnection } = partner
+  return (
+    <Box mt={126}>
+      <GridColumns mt={6} gridRowGap={[2, 4]}>
+        <Column span={3}>Filters would be here</Column>
+        <Column span={9}>
+          <ArtworkGrid
+            artworks={artworksConnection}
+            columnCount={[2, 2, 3, 3]}
+            preloadImageCount={0}
+          />
+        </Column>
+      </GridColumns>
+    </Box>
```

Comment:
Rather than doing this manually; there's a component that handles this already. Take a look at https://github.com/artsy/force/pull/7313/commits/5c3a4a92fec87379c3a08c04916ee64131a0e87c where I just added this to the new Gene page.